src/compound/SupplyVaultBase.sol

```diff
@@ -19,18 +19,34 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
     using CompoundMath for uint256;
     using SafeTransferLib for ERC20;
 
+    /// EVENTS ///
+
+    /// @notice Emitted when a new rewards `recipient` is set.
+    /// @param recipient The recipient of the rewards.
+    event RewardsRecipientSet(address recipient);
+
+    /// @notice Emitted when rewards rewards are transferred to `recipient`.
+    /// @param recipient The recipient of the rewards.
+    /// @param amount The amount of rewards transferred.
+    event RewardsTransferred(address recipient, uint256 amount);
+
     /// ERRORS ///
 
     /// @notice Thrown when the zero address is passed as input.
     error ZeroAddress();
```

Comment:
```suggestion
    error AddressZero();
```

Reply:
I think both can be used. I often see the zero address

Reply:
Ok. In classic language you would say address zero, that is why I proposed.

Reply:
https://coinacademy.fr/lexique/zero-address/ but I can change

Reply:
Oh yes I'm probably wrong